site-config/lungmap/dev/index/filesEntityConfig.ts

```diff
@@ -0,0 +1,275 @@
+        } as ComponentConfig<typeof C.NTagCell, FilesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.NUCLEIC_ACID_SOURCE,
+        id: HCA_DCP_CATEGORY_KEY.NUCLEIC_ACID_SOURCE,
+        width: { max: "1fr", min: "120px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildAggregatedProtocolPairedEnd,
+        } as ComponentConfig<typeof C.BasicCell, FilesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.PAIRED_END,
+        id: HCA_DCP_CATEGORY_KEY.PAIRED_END,
+        width: { max: "1fr", min: "120px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorOrganismAge,
+        } as ComponentConfig<typeof C.NTagCell, FilesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.ORGANISM_AGE,
+        id: HCA_DCP_CATEGORY_KEY.ORGANISM_AGE,
+        width: { max: "1fr", min: "120px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildAggregatedDonorBiologicalSex,
+        } as ComponentConfig<typeof C.BasicCell, FilesResponse>,
+        header: "Sex",
+        id: HCA_DCP_CATEGORY_KEY.BIOLOGICAL_SEX,
+        width: { max: "1fr", min: "120px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedSpecimenDisease,
+        } as ComponentConfig<typeof C.NTagCell, FilesResponse>,
+        header: "Disease (Specimen)",
+        id: HCA_DCP_CATEGORY_KEY.SPECIMEN_DISEASE,
+        width: { max: "1fr", min: "120px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorDisease,
+        } as ComponentConfig<typeof C.NTagCell, FilesResponse>,
+        header: "Disease (Donor)",
+        id: HCA_DCP_CATEGORY_KEY.DONOR_DISEASE,
+        width: { max: "1fr", min: "120px" },
+      },
+      {
+        columnVisible: false,
+        componentConfig: {
+          component: C.NTagCell,
+          viewBuilder: V.buildAggregatedDonorDevelopmentStage,
+        } as ComponentConfig<typeof C.NTagCell, FilesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.DEVELOPMENT_STAGE,
+        id: HCA_DCP_CATEGORY_KEY.DEVELOPMENT_STAGE,
+        width: { max: "1fr", min: "148px" },
+      },
+      {
+        componentConfig: {
+          component: C.BasicCell,
+          viewBuilder: V.buildTotalCells,
+        } as ComponentConfig<typeof C.BasicCell, FilesResponse>,
+        header: HCA_DCP_CATEGORY_LABEL.CELL_COUNT,
+        id: HCA_DCP_CATEGORY_KEY.CELL_COUNT,
+        width: { max: "1fr", min: "124px" },
+      },
+    ],
+    defaultSort: {
+      desc: SORT_DIRECTION.ASCENDING,
+      id: HCA_DCP_CATEGORY_KEY.FILE_NAME,
+    },
+  } as ListConfig<FilesResponse>,
+  listView: {
```

Comment:
Hi @jpaten would you mind removing `listView` from `filesEntityConfig`. Thank you!